packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -69,7 +69,10 @@ export const LOCAL_MEDIA_PER_PAGE = 50;
 
 function MediaUpload({ render: _render, onSelect }) {
   const open = () => {
-    const image = { type: 'image', src: 'https://www.example.com/media1' };
+    const image = {
+      type: 'image',
+      src: 'http://localhost:9876/__static__/saturn.jpg',
+    };
```

Comment:
Tests are failing everywhere due to `https://www.example.com/media1`. The `<ImageDisplay />` component was trying to fetch that url when using the replace media fixture but that url isn't a place where an image can be downloaded.